File: cassandra/go/write.go
package main

import (
	"log"

	"github.com/gocql/gocql"
)

const (
	host     = "cassandra-3b8d4ed6-myfirstcloudhub.aivencloud.com"
	port     = 15193
	username = "avnadmin"
	password = "<your password here>"
)

// *******
// DISCLAIMER: This code might fail because of "unable to connect to initial hosts: unexpected authenticator"
// We have an open pull request at github.com/gocql/gocql for them to add the Aiven authenticator to the approved list
// ********

// Create a keyspace called 'example_keyspace' before running this

func main() {
	cluster := gocql.NewCluster(host)
	cluster.Port = port
	cluster.ProtoVersion = 4
	cluster.Authenticator = gocql.PasswordAuthenticator{
		Username: username,
		Password: password,
	}
	cluster.SslOpts = &gocql.SslOptions{
		CaPath: "ca.pem",
	}
	cluster.Keyspace = "example_keyspace"
	cluster.Consistency = gocql.Quorum
	session, err := cluster.CreateSession()
	if err != nil {
		log.Fatal(err)
	}
	defer session.Close()
	if err := session.Query(
		"CREATE TABLE example_table (id int PRIMARY KEY, message text)",
	).Exec(); err != nil {
		log.Print(err)
	}
	if err := session.Query(
		"INSERT INTO example_table (id, message) VALUES (?, ?)", 123, "Hello world!",
	).Exec(); err != nil {
		log.Fatal(err)
	}
	log.Println("Row inserted")
}

File: cassandra/python/write.py
import ssl
from cassandra.cluster import Cluster
from cassandra.auth import PlainTextAuthProvider

import cassandra

host = "cassandra-3b8d4ed6-myfirstcloudhub.aivencloud.com"
port = 15193
auth_provider = PlainTextAuthProvider("avnadmin", "nr0dfnswz36xs9pi")
ssl_options = {"ca_certs": "ca.pem", "cert_reqs": ssl.CERT_REQUIRED}

with Cluster([host], port=port, ssl_options=ssl_options, auth_provider=auth_provider) as cluster:
    with cluster.connect() as session:
        try:
            session.execute("CREATE KEYSPACE example_keyspace WITH REPLICATION = {'class': 'NetworkTopologyStrategy', 'aiven': 3}")
        except cassandra.AlreadyExists as ex:
            print(ex)
        session.execute("USE example_keyspace")
        try:
            session.execute("CREATE TABLE example_table (id int PRIMARY KEY, message text)")
        except cassandra.AlreadyExists as ex:
            print(ex)
        session.execute("INSERT INTO example_table (id, message) VALUES (%s, %s)", (123, "Hello world!"))
        print("Row inserted")
